### PR TITLE
Initial PR for the project, first version of scraping code

### PR DESCRIPTION
**Pull request by nrosed**
Initial PR for the project. Here I have only the initial version of the scraper, most of the new code is boiler plate, the only files I wrote (apart from gitignore / readme / environment / license) are:

- /nature_news_scraper/run_scrape_gold_standard.sh
- /nature_news_scraper/nature_news_scraper/spiders/precrawl.py- 
- /nature_news_scraper/nature_news_scraper/spiders/article_crawl.py

### CODE REVIEW

File: README.md
Comment (danich1) [inline README.md:15]: Are you going to scale up to all articles between 2010-2020?

File: README.md
Comment (danich1) [inline README.md:11]: Really cool!! Didn't know this was out there. Definitely looks like a cool package to use.

File: environment.yml
Comment (danich1) [inline environment.yml:5]: This type of file can change based on personal preference. Some (including me) only want it to contain packages that need to be installed rather than all at once. Others like to have it this way ass it guarantees that there wont be an versioning issues down the road. I won't suggest any changes to this file as I'll assume you are in the non-me category :)

File: nature_news_scraper/run_scrape_gold_standard.sh
Comment (danich1) [inline nature_news_scraper/run_scrape_gold_standard.sh:3]: How big is this links.json file? Also, is this file being used to crawl the news articles? If so, then I may recommend adding this to the repository instead of ignoring it to help with reproducibility (Can come in future PR. If there is a size issue feel free to ping me, dealt with this kind of headaches all the time).

**Review by danich1 — APPROVED** [review]
Looks good to me (LGTM)! Have a few questions that once addressed feel free to merge. Glossed over the auto generated files this time as I assume they will be changed in the future.

File: README.md
Comment (nrosed) [reply README.md:15]: Eventually I will scale up all articles from 2001 - 2020. This is just to make a benchmark dataset that I will hand annotate.

File: README.md
Comment (nrosed) [reply README.md:11]: yeah! I found it easy to use and it's well documented, (my husband pointed it out to me). I am a super novice in scraping so I needed something that was gonna almost do it all for me.

File: nature_news_scraper/run_scrape_gold_standard.sh
Comment (nrosed) [reply nature_news_scraper/run_scrape_gold_standard.sh:3]: This is a good point. It is not big and since it'll be used for the benchmark I think this is a good idea.

File: environment.yml
Comment (nrosed) [reply environment.yml:5]: I don't have a strong opinion here, I just have experienced some r-package dependency nightmares... but I could be persuaded in either direction.

Comment (danich1) [reply environment.yml:5]: I say leave it as I have dealt with quite a bit of conda dependency headaches myself. Better for things to work than break and have to figure it out down the road.

**Pull request merged**
